@xen-orchestra/proxy/docs/api.md

```diff
@@ -49,6 +51,24 @@ Content-Type: application/json
 {"id":"task5"}
 ```
 
+Or
```

Comment:
Give a different name between those two JSON formats (see our conversation).

Also, put the simplest format first.